src/sentry/integrations/client.py

```diff
@@ -193,37 +197,44 @@ def _request(
             sample_rate=1.0,
             tags={"integration": self.integration_name},
         )
-        try:
-            resp = getattr(session, method.lower())(
-                url=full_url,
-                headers=headers,
-                json=data if json else None,
-                data=data if not json else None,
-                params=params,
-                auth=auth,
-                verify=self.verify_ssl,
-                allow_redirects=allow_redirects,
-                timeout=timeout,
-            )
-            resp.raise_for_status()
-        except ConnectionError as e:
-            self.track_response_data("connection_error", e)
-            raise ApiHostError.from_exception(e)
-        except Timeout as e:
-            self.track_response_data("timeout", e)
-            raise ApiTimeoutError.from_exception(e)
-        except HTTPError as e:
-            resp = e.response
-            if resp is None:
-                self.track_response_data("unknown", e)
-                self.logger.exception(
-                    "request.error", extra={"integration": self.integration_name, "url": full_url}
+
+        with sentry_sdk.start_span(
+            op="integrations.http",
+            transaction=u"integrations.http_response.{}".format(self.integration_name),
```

Comment:
Does this new sub-transaction end up inheriting the traceid from the parent transaction? I don't remember if we got that part working when we last chatted.

Reply:
yeah Search By Trace links to the other api transactions, the trick was to not use the Span constructor I believe